air-script/tests/constants/constants.rs

```diff
@@ -0,0 +1,104 @@
+use winter_air::{Air, AirContext, Assertion, AuxTraceRandElements, EvaluationFrame, ProofOptions as WinterProofOptions, TransitionConstraintDegree, TraceInfo};
+use winter_math::{fields, ExtensionOf, FieldElement};
+use winter_utils::{collections, ByteWriter, Serializable};
+
+const A: u64 = 1;
+const B: Vec<u64> = vec![0, 1];
+const C: Vec<Vec<u64>> = vec![vec![1, 2], vec![2, 0]];
```

Comment:
> Also, we should declare these as Felt' rather than u64 types (via Felt::new()).
Question: Since we might need these in base field or extension field depending on where they are used, how should that be handled if we store these as felts here?

Reply:
We can always convert from base field element to an extension field element. Basically, `E::from(Felt)` should work just as well. In case `E` is the base field, `E:from(Felt)` would be free.